### ISSUE TITLE
Synthetic Proj construction?

### ISSUE THREAD

**Issue by xuanruiqi** [felixwellen/synthetic-zariski#15]
It would be good to have a synthetic version of the Proj construction, as it would be useful for constructing projective schemes.

**Comment by felixwellen** [comment]
Yes that would be nice. Also to have a correspondence of schemes of the form $\mathrm{Proj}(S)$ to closed subsets of $\mathbb{P}^n$ as defined in [foundations](https://felix-cherubini.de/iag.pdf), "6 Projective Space". Maybe some of the things @mnieper wrote in [random facts](https://felix-cherubini.de/random.pdf) under "2 Homogeneity" can be used. Note that - as far as I know - it is also still open if every closed subset of $\mathbb P^n$ is a common zero set of homogeneous polynomials.

**Comment by iblech** [comment]
We at least have [Definition 18.13 of my thesis](https://rawgit.com/iblech/internal-methods/master/notes.pdf#page=159). I don't know whether the description can be simplified.

**Comment by xuanruiqi** [comment]
Thanks @iblech! I think that should work. Sorry if this is a stupid question, but what kind of quotient did you mean by the slash in Def. 18.13?

**Comment by iblech** [comment]
When now rereading that passage, I also noticed that I should have been clearer!

I was referring to a version of the usual rescaling relation: Homomorphisms $\alpha,\beta : A \to R[T]$ are equivalent iff there is an invertible number $\lambda : R$ such that for all natural numbers $n$ and all homogenuous elements $x$ of degree $n$, $\alpha(x) = \lambda^n \beta(x)$.

**Comment by felixwellen** [comment]
Maybe the question was also what quotients are in the internal language?
I know how to answer that in HoTT: We would use a set-quotient (6.10 in the HoTT-book).

**Comment by iblech** [comment]
Ah, yes! In my thesis I was working in the internal universe of the 1-topos, so everything was set-truncated, so yes, in the context of HoTT I mean the set quotient.

**Comment by xuanruiqi** [comment]
Thanks for clarifying! Yes, everything here seem to be sets at least as far as I know of

**Comment by mnieper** [comment]
Now that the winter term is coming to an end and I have some time, I am going to revisit this again.

**Comment by mnieper** [comment]
There is already some stuff in the branch called "projectivity". I still have to write a number of sections.

**Comment by xuanruiqi** [comment]
Thanks! Let's talk about this in Gothenburg.
